test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,51 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.*;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+
+public class UtilsTest {
+
+	private Utils utils;
+	
+	@Before
+	public void setup() {
+		utils = new Utils();
+	}
+	
+	@Test
+	public void check_null_object() {
```

Comment:
Note the naming convention for test methods (the first part is feature/method under test), e.g. `isAnyNull_emptyList_falseReturned`